src/components/screens/FinishPost/FinishPost.js

```diff
@@ -6,23 +6,24 @@ import ConfirmButton from "../../common/ConfirmButton";
 import HeaderTitle from "../../common/HeaderTitle";
 import InputText from "../../common/InputText";
 import TextArea from "../../common/TextArea";
-import { URI } from "../../../constants/URI";
 import { BACKGROUND_COLOR } from "../../../constants/colors";
+import type { Uri } from "../../../types/types";
 
 type Props = {
-  navigator: Object
+  navigator: Object,
+  uri: Uri
 };
 
 class FinishPost extends PureComponent<Props> {
-  _previousPage = () => {
+  previousPage = () => {
```

Comment:
functions names must start with verbs or prefixes like `on, is, has...`. In this case, there should be a verb, that represents the function meaning, for example `goToPreviousPage` or `navigateToPreviousPage`